src/pages/BookmarksPage.js

```diff
@@ -4,31 +4,33 @@ import Content from '../components/Content.js';
 import DiveWish from '../components/DiveWish.js';
 
 export default function BookmarksPage({
-  bookmarkedWishes,
+  bookmarkedPosts,
   onToggleBookmark,
   onToggleCheckmark,
-  onEditDiveWish,
-  onDeleteDiveWish,
+  onEditPost,
+  onDeletePost,
 }) {
   return (
     <>
       <Header>Your favorites</Header>
       <Content>
         <Grid>
-          {bookmarkedWishes.length > 0 ? (
-            bookmarkedWishes.map(wish => (
-              <DiveWish
-                key={wish.id}
-                destination={wish.destination}
-                notes={wish.notes}
-                isBookmarked={wish.isBookmarked}
-                isArchived={wish.isArchived}
-                onToggleBookmark={() => onToggleBookmark(wish.id)}
-                onToggleCheckmark={() => onToggleCheckmark(wish.id)}
-                onEditDiveWish={() => onEditDiveWish(wish)}
-                onDeleteDiveWish={() => onDeleteDiveWish(wish.id)}
-              />
-            ))
+          {bookmarkedPosts?.length > 0 ? (
+            bookmarkedPosts
+              .reverse()
```

Comment:
I looked it up: `.reverse()` mutates the original array [(see here)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse)